modules/ROOT/pages/dataweave-indexed-readers.adoc

```diff
@@ -0,0 +1,51 @@
+= Indexed Readers in DataWeave
```

Comment:
Is this topic really about indexed readers or is it about the various methods that DataWeave uses to process input data that is too large for memory?